kratos/python/add_containers_to_python.cpp

```diff
@@ -81,6 +81,13 @@ template< class TBinderType, typename TContainerType, typename TVariableType > v
     binder.def("Clear", [](TContainerType& container){container.Clear();} );
 }
 
+template< class TBinderType, typename TContainerType, typename TVariableType > void DataValueContainerIndexingUtility(TBinderType& binder)
+{
+    // Data value container
+    VariableIndexingUtility<TBinderType, TContainerType, TVariableType>(binder);
+    binder.def("Erase", [](TContainerType& container, const TVariableType& rV){return container.Erase(rV);} );
```

Comment:
Couldn't this be added to the original VariableIndexingUtility ?

Reply:
Because not all classes have Erase (that was my first mistake)

Reply:
For example: VariablesListDataValueContainer does not have Erase

Reply:
The merge is blocked @maceligueta

Reply:
Well, I think we should wonder why some objects which have `GetValue`, `SetValue`, `Has `and `Clear `do not have `Erase`. Maybe it's time to add it to those objects, no?. Not necessarily you, @loumalouomega , but it sounds weird or even like a pending TODO in my opinion.

Reply:
Do you remember which objects were those?

Reply:
VariablesListDataValueContainer for example

Reply:
Does anybody know why these lines are commented out? https://github.com/KratosMultiphysics/Kratos/blob/9b99868feea277b70dafe9c74996eff5bc13d429/kratos/containers/variables_list_data_value_container.h#L429
@pooyan-dadvand @RiccardoRossi

Reply:
No idea

Reply:
I should have deleted it. The method comes from Kratos v0.

You cannot erase a VariablesListData because this implies remove it from all the nodes that sharing the same variables list and is practically impossible.